Source/Android/src/org/dolphinemu/dolphinemu/gamelist/GameListFragment.java

```diff
@@ -123,11 +124,29 @@ public void onListItemClick(ListView listView, View view, int position, long id)
 
 		// Show a toast indicating which game was clicked.
 		Toast.makeText(getActivity(), String.format(getString(R.string.file_clicked), item.getPath()), Toast.LENGTH_SHORT).show();
+		
+		Notification note=new Notification(R.drawable.stat_notify_chat,
+                                          "Dolphin Emulator",
+                                          System.currentTimeMillis());
+		
+		Intent i=new Intent(this, GameListActivity.class);
+    
+		i.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP|
+				Intent.FLAG_ACTIVITY_SINGLE_TOP);
+		
+		PendingIntent pi=PendingIntent.getActivity(this, 0,
+                                                  i, 0);
+      
+		note.setLatestEventInfo(this, "Dolphin Emulator",
+                              "Running",
+                              pi);
+		note.flags|=Notification.FLAG_NO_CLEAR;
 
 		// Start the emulation activity and send the path of the clicked ROM to it.
 		Intent intent = new Intent(getActivity(), EmulationActivity.class);
 		intent.putExtra("SelectedGame", item.getPath());
 		startActivity(intent);
+		startForeground(1337, note);
```

Comment:
Due to lack of memory GamelistActivity is killed. I'm trying to make a GamelistActivity Service for delay / prevent killing by LMK.